basis/db/tuples/tuples.factor

```diff
@@ -63,6 +67,10 @@ GENERIC: eval-generator ( singleton -- object )
     [ <insert-user-assigned-statement> ] cache
     [ bind-tuple ] keep execute-statement ;
 
+: do-each-tuple ( exemplar-tuple statement quot -- tuples )
```

Comment:
Might be nice to show ``quot`` stack effect, ``quot: ( tuple -- )``.

Reply:
Will do, thanks!